WalletWasabi/WabiSabi/Client/UnexpectedRoundPhaseException.cs

```diff
@@ -0,0 +1,22 @@
+using NBitcoin;
+using System.Collections;
+using WalletWasabi.WabiSabi.Backend.Rounds;
+using WalletWasabi.WabiSabi.Models;
+
+namespace WalletWasabi.WabiSabi.Client;
+
+public class UnexpectedRoundPhaseException : Exception
```

Comment:
I would rename this to `UnexpectedRoundPhaseChangeException`

Reply:
It is OK IMO.